src/Command/Checker.php

```diff
@@ -94,8 +96,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln('<bg=green;fg=white>    Everything is OK!     </>');
     }
 
-    private function docIsCompatible($headerValidator, $fileContent)
+    private function docIsCompatible($headerValidator, $fileContent, $docheaderFile)
     {
-        return $headerValidator->__invoke($fileContent) || false !== strpos($fileContent, $this->header);
+        return $headerValidator->__invoke($fileContent) || false !== strpos($fileContent, $docheaderFile);
+    }
+
+    private function getDocheaderFileContent(InputInterface $input)
+    {
+        $docheaderFile = $input->getOption('docheader');
+
+        if ('.docheader' === $docheaderFile) {
+            $docheaderFile = getcwd() . '/' . $docheaderFile;
+        }
+
+        $docheader = (new DocheaderFileResolution())->__invoke($docheaderFile);
```

Comment:
This could be more simply written as `(new DocheaderFileResolution())($docheaderFile)`.